inst/extdata/PA2022CH_en_exec_summary/template.Rmd

```diff
@@ -6,6 +6,7 @@ always_allow_html: true
 params:
   survey_dir: NULL
```

Comment:
seems like this `survey_dir` might not be relevant/available anymore?

Reply:
still relevant, `score_card_dir` is only used for the score card section, not for the survey section (that still uses `survey_dir`)

Reply:
ok but, didn't you remove the `survey_dir` from the function argument that triggers the render of the Rmds? So how would it know what `survey_dir` is?

I trust you, so I'm gonna approve now, just please consider it, because it's not obvious to me.